docs/build-provenance-for-all-registries.md

```diff
@@ -0,0 +1,104 @@
+# Build Provenance for All Registries
```

Comment:
IMO "registry" isn't a universal term. Maybe "package registries", "package repositories", or just make sure to define it in the first paragraph or two

Reply:
Package registries sounds good to me! I have updated the text.